src/app/pages/PerpetualPage/hooks/usePerpetual_executeTransaction.ts

```diff
@@ -52,6 +52,7 @@ export const usePerpetual_executeTransaction = (useGSN: boolean) => {
             const tradeTx: PerpetualTxTrade = transaction;
             return trade(
               tradeTx.isClosePosition,
+              tradeTx.keepPositionLeverage,
```

Comment:
Ah shit I'll have to fix some conflicts with my limit order changes branch, as I fully reworked the executeTransaction hook and removed the individual transaction hooks :grimacing:

Not your problem though :sweat_smile: